module/updatable_configs/require_approvals_for_seal_construction.go

```diff
@@ -0,0 +1,63 @@
+package updatable_configs
+
+import (
+	"sync"
+
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/module/defaults"
+	"github.com/onflow/flow-go/module/updatable_configs/validation"
+)
+
+// min number of approvals required for constructing a candidate seal
+type RequiredApprovalsForSealConstructionInstance struct {
+	sync.RWMutex
+	requiredApprovalsForSealConstruction uint
+}
+
+var createRequiredApprovalsForSealConstructionOnce sync.Once
+var instanceRequiredApprovalsForSealConstruction *RequiredApprovalsForSealConstructionInstance
+
+func createRequiredApprovalsForSealConstruction() *RequiredApprovalsForSealConstructionInstance {
```

Comment:
I personally find that the method name is over-specific. It says it creates the instance, but that is only true for the first time. How about we rename it to `acquireRequiredApprovalsForSealConstruction` ?